source/about/mattermost-mobile-releases.md

```diff
@@ -18,7 +18,7 @@ See the [Mobile app changelog](/about/mobile-app-changelog) for release details,
 
 | **Release** | **Released on** | **Support ends** | **Compatible with** |
 |:---|:---|:---|:---|
-| v2.20 {ref}`FEATURE <release-types>` [Download](https://github.com/mattermost/mattermost-mobile/releases/tag/v2.20.0) \| {ref}`Changelog <release-v2-20-0>` | 2024-09-16 | 2024-10-15 | {ref}`v10.0 <release-v10-0-major-release>`, {ref}`v9.11 <release-v9-11-extended-support-release>`, {ref}`v9.10 <release-v9-10-feature-release>`, {ref}`v9.9 <release-v9-9-feature-release>`, {ref}`v9.5 <release-v9-5-extended-support-release>` |
+| v2.20 {ref}`FEATURE <release-types>` [Download](https://github.com/mattermost/mattermost-mobile/releases/tag/v2.20.0) \| {ref}`Changelog <release-v2-20-0>` | 2024-09-16 | 2024-10-15 | {ref}`v10.0 <https://docs.mattermost.com/about/mattermost-v10-changelog.html#release-v10-0-major-release>`, {ref}`v9.11 <release-v9-11-extended-support-release>`, {ref}`v9.10 <release-v9-10-feature-release>`, {ref}`v9.9 <release-v9-9-feature-release>`, {ref}`v9.5 <release-v9-5-extended-support-release>` |
```

Comment:
The link wasn't working because it was malformed (a dash instead of a period): ``(release-v10.0-major-release)=``

Reply:
Thank you!